### ISSUE TITLE
How can import in IDE

### ISSUE THREAD

**Issue by federicolucca** [Xinyuan-LilyGO/T-solar#4]
Goodmorning,
How can import this project in a IDE? Arduino IDE/Visual Studio ecc

Thanks
Federico

**Comment by msam-rak** [comment]
https://www.iar.com/products/architectures/st/iar-embedded-workbench-for-stm8/
https://www.st.com/en/development-tools/stm8-ides.html

**Comment by bakwc** [comment]
Have you managed to get it work?
Does it support battery charge from solar panels?

**Comment by mtuxpe** [comment]
> Have you managed to get it work? Does it support battery charge from solar panels?

I compiled the github example using IAR but I don't know how to flash the project.
I haven't tested a solar panel.

**Comment by bakwc** [comment]
Have you tried ST-Link?

**Comment by mtuxpe** [comment]
> Have you tried ST-Link?

Yes. Original ST-link but I don't know how to connect the board on ST-Link
